readme.md

```diff
@@ -20,19 +20,24 @@ Python 3.10 or later should be installed.
 
 ## Setup instructions
 1. Make sure you have Weaviate installed and set up. [Check out the documentation](https://weaviate.io/developers/weaviate/current/installation/index.html) for more information!
-2. Run the docker file 
+2. Install the Python packages
+   ```bash
+    pip install -r requirements.txt
+    pip install -U "git+https://github.com/weaviate/weaviate-python-client.git@v3.99.0a3#egg=weaviate-client[GRPC]"
```

Comment:
I suggest moving this to `requirements.txt`. It would involve adding:

```
git+https://github.com/weaviate/weaviate-python-client.git@v3.99.0a3#egg=weaviate-client[GRPC]
```

As an additional line.